earthkit/geo/figure.py

```diff
@@ -55,7 +46,7 @@ def scale(self, *args):
             return args[0]
 
 
-IFS_SPHERE = IFSSphere()
+IFS_SPHERE = Sphere(6371229)
```

Comment:
Should this be appended ".0" so that it is clear to be a float? (It's more of a semantic than a pedantic comment, as  you've corrected the unit sphere above)

Reply:
Good point. I have changed it.